collect_app/src/main/java/org/odk/collect/android/widgets/SpinnerMultiWidget.java

```diff
@@ -220,4 +214,14 @@ public void onClick(DialogInterface dialog, int which,
         AlertDialog alert = alertBuilder.create();
         alert.show();
     }
+
+    private void showSelectedValues(List<String> selectedValues) {
+        if (selectedValues.size() > 0) {
```

Comment:
How about if (....**isEmpty()**) clearAnswer() else the other

Reply:
That will also do, but i don't think it will be an improvement. The call to the ```size()``` method is also ```O(1)```.  
Any reason why calling ```isEmpty``` is preferable ? For readability ?

Reply:
https://stackoverflow.com/questions/1508975/why-is-list-size0-slower-than-list-isempty-in-java#1508993

Reply:
Thank you :+1:

Reply:
I was gonna give another reason, but it’s already in the page @shobhitagarwal1612 linked to:
>isEmpty is a clearer definition of what it is you actually care about and are testing, and so makes your code a bit more easily understandable.

Also, `size() == 0` is 6 tokens (for the human to parse) and `isEmpty()` is 3.

(Scala adds `nonEmpty()` which I use quite a lot.)